### PR TITLE
feat: plugin rework

### PR DESCRIPTION
**Pull request by vonabarak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+    модуль, а после двоеточия объект внутри модуля.
+   
+    Функциональность бота можно расширять при помощи плагинов. Список включеных плагинов
+    задаётся настройкой `PLUGINS`, которая является списком строк в описанном выше
+    формате. Имя класса `Plugin` можно не указывать.
+    Плагины вкючённые в библиотеку:
+   
+    | Путь                             | Описание
+    -----------------------------------|:-----------------------------------
+    | boxv2.plugins.logger             | расширенне логирование (Loguru)
```
Comment (lxmnk) [inline README.md:73]: Расшиненн**о**е

File: boxv2/error_handlers/server_unknown.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"""Definition for error handler for message from unregistered CTS."""
+
+from botx import BotDisabledErrorData, BotDisabledResponse, UnknownBotError
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+from starlette.status import HTTP_503_SERVICE_UNAVAILABLE
+
+
+async def message_from_unknown_server_handler(
```
Comment (lxmnk) [inline boxv2/error_handlers/server_unknown.py:9]: This exception handler doesn't work, you can safely drop it.

File: boxv2/main.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+@click.command()
+@click.option("--verbose", "-v", is_flag=True)
+@click.option("--plugin", "-p", multiple=True, default=["debug", "logger"])
+@click.option("--template", "-t", required=False)
+@click.argument("bot_name", nargs=1, type=click.Path(file_okay=False, dir_okay=False))
+def main(plugin: list[str], template: Optional[str], bot_name: str, verbose: bool) -> None:
+    """Create a bot project."""
+    globals()["VERBOSE"] = verbose
```
[the remark this answers is not in the document]
Comment (vonabarak) [reply boxv2/main.py]: replaced with logger

File: boxv2/plugin.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+from boxv2.settings import BaseAppSettings
+from boxv2.utils.import_utils import import_object
+
+
+class BasePluginSettings(BaseSettings):
+    """Main settings for splitting."""
+
+    class Config:  # noqa: WPS431
+        env_file = ".env"
+        json_loads = lambda x: x  # noqa: E731,WPS111
```
[the remark this answers is not in the document]
Comment (vonabarak) [reply boxv2/plugin.py:20]: Pydantic tries to read .env file as json without this function.

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+pytest-timeout = "~1.3.4"
+asgi-lifespan = "~1.0.1"
+molten = "~1.0.2"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.scripts]
+boxv2 = 'boxv2.main:main'
```
[the remark this answers is not in the document]
Comment (vonabarak) [reply pyproject.toml]: fxd

File: README.md [hunk 6]
```diff
@@ -1,2 +1,73 @@
...
+    ```
+   
+3. Установка зависимостей
+
+    ```bash
+    poetry install
+    ```
+    Важно: библиотека boxv2 в созданном проекте автоматически обновится из гита до
+    последней версии. Если такое поведение нежелательно, то необходимо указать
+    в файле pyproject.toml конкретную версию
```
[the remark this answers is not in the document]
Comment (vonabarak) [reply README.md:32]: done

File: boxv2/plugins/debug/plugin.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
+"""Debug plugin."""
+
+from typing import Any
+
+from boxv2.plugin import BasePlugin
+from boxv2.plugins.debug.commands import collector
+
+
+class Plugin(BasePlugin):
```
[the remark this answers is not in the document]
Comment (vonabarak) [reply boxv2/plugins/debug/plugin.py]: done

**Review by lxmnk — APPROVED** [review]

**Review by Nimond — APPROVED** [review]

**Pull request merged**